src/actions/forecast.actions.js

```diff
@@ -0,0 +1,20 @@
+import { forecastConstants } from '../constants';
+import { forecastPeriodService } from '../services';
+
+
+export const forecastActions = {
+	fetchForecast,
+};
+
+function fetchForecast(){
+    return async (dispatch,currentState)=>{
```

Comment:
better to use `getState`. I think it is more convenient

Reply:
+

Reply:
+